docs/pages/components/Notice.mdx

```diff
@@ -76,14 +87,39 @@ Icon Type](/components/icon).
 </Notice>
 ```
 
+### Dismissable
+
+```jsx example
+function DismissableNotice() {
+  const [dismissed, setDismissed] = useState(false);
+  const handleDismiss = () => {
+    setDismissed(true);
+    setTimeout(() => setDismissed(false), 2000);
+  };
+
+  return dismissed ? null : (
+    <Notice type="info" title="Information" dismissable>
```

Comment:
```suggestion
    <Notice type="info" title="Information">
```

Dismissable doesn't appear to be a valid prop.

Reply:
Oof, I forgot Copilot added that in when tabbing through 🤣